modules/admin-top-bar/module.php

```diff
@@ -0,0 +1,130 @@
+<?php
+namespace Elementor\Modules\AdminTopBar;
+
+use Elementor\Core\Base\App as BaseApp;
+use Elementor\Core\Experiments\Manager;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseApp {
+	//private $elementor_pages = [];
+	const ELEMENTOR_MENU_ITEM_IDS = [
+		'toplevel_page_elementor',
+		'menu-posts-elementor_library'
+
+	];
```

Comment:
Why do you need it on the server side?

Reply:
When a developer will add new page, it will be more easy for him to see straight in the module the classes which the top bar can be added, and if he need to add new one, he will do it in the module.

Reply:
But it's not related to the server, so, please move it to the main App.

Reply:
Done.